client/my-sites/plugins/main.jsx

```diff
@@ -567,12 +251,15 @@ export default React.createClass( {
 		return some( this.props.sites.getSelectedOrAllWithPlugins(), site => site && site.jetpack && site.canUpdateFiles );
 	},
 
-	updateAllPlugins() {
-		PluginsActions.removePluginsNotices( this.state.notices.completed.concat( this.state.notices.errors ) );
-		this.state.plugins.forEach( plugin => {
-			plugin.sites.forEach( site => PluginsActions.updatePlugin( site, site.plugin ) );
-		} );
-		this.recordEvent( 'Clicked Update all Plugins', true );
+	showPluginListPlaceholders( isWpCom ) {
+		let showPlaceholders = isEmpty( this.state.plugins );
+		const selectedSite = this.props.sites.getSelectedSite();
+
+		if ( showPlaceholders && selectedSite && selectedSite.jetpack === isWpCom ) {
+			showPlaceholders = false;
+		}
+
+		return showPlaceholders;
```

Comment:
Since this is just a boolean, we can skip the conditional and just return the condition.

```js
shouldShowPluginListPlaceholders( isWpCom ) {
	const { plugins } = this.state;
	const { sites } = this.props;

	return ! isEmpty( plugins ) && isWpCom === get( sites.getSelectedSite(), 'jetpack' );
}
```